mimic/rest/auth_api.py

```diff
@@ -345,6 +345,120 @@ def validate_token(self, request, token_id):
+                    "id": "hybrid:123456",
+                    "name": "hybrid:123456",
+                }
+                response["access"]["user"] = {
+                    "id": "34",
+                    "name": "HybridThreeFour",
+                    "roles": [{"id": "1",
+                               "name": "monitoring:creator",
+                               "description": "Monitoring Creator"},
+                              {"id": "2",
+                               "name": "creator",
+                               "description": "Creator"}],
+                    "RAX-AUTH:contactId": "34"
+                }
+
+        if token_id in get_presets["identity"]["dedicated_limited_device_permission_holder"]:
+                response["access"]["token"]["tenant"] = {
+                    "id": "hybrid:123456",
+                    "name": "hybrid:123456",
+                }
+                response["access"]["user"] = {
+                    "id": "56",
+                    "name": "HybridFiveSix",
+                    "roles": [{"id": "1",
+                               "name": "monitoring:observer",
+                               "description": "Monitoring Observer"},
+                              {"id": "2",
+                               "name": "observer",
+                               "description": "Observer"}],
+                    "RAX-AUTH:contactId": "56"
+                }
+
+        if token_id in get_presets["identity"]["dedicated_other_account_observer"]:
+                response["access"]["token"]["tenant"] = {
+                    "id": "hybrid:654321",
+                    "name": "hybrid:654321",
+                }
+                response["access"]["user"] = {
+                    "id": "78",
+                    "name": "HybridSevenEight",
+                    "roles": [{"id": "1",
+                               "name": "monitoring:observer",
+                               "description": "Observer"},
+                              {"id": "2",
+                               "name": "observer",
+                               "description": "Observer"}],
+                    "RAX-AUTH:contactId": "78"
+                }
+
+        if token_id in get_presets["identity"]["dedicated_other_account_admin"]:
+                response["access"]["token"]["tenant"] = {
+                    "id": "hybrid:654321",
+                    "name": "hybrid:654321",
+                }
+                response["access"]["user"] = {
+                    "id": "90",
+                    "name": "HybridNineZero",
+                    "roles": [{"id": "1",
+                               "name": "monitoring:admin",
+                               "description": "Admin"},
+                              {"id": "2",
+                               "name": "admin",
+                               "description": "Admin"}],
+                    "RAX-AUTH:contactId": "90"
+                }
+
```

Comment:
The concern is valid.  In fact I'd go as far as to say that removing fixtures entirely from both these APIs and having test suites load their own data would be preferable.  But that requires the work of adding ingestion endpoints to both API's (I strongly considered doing this), or learning/creating some other mechanism for loading data (does one exist? I'm not a Mimic expert).

For now I think I need this expediency.  If the Mimic maintainers reject this, I'll likely just have to run a MaaS fork of Mimic until I can find some time to do the job right.